.gitignore

```diff
@@ -18,3 +18,8 @@ yarn-debug.log*
 yarn-error.log*
 yarn.lock
 !/yarn.lock
+
+site/node_modules
+site/public
+site/db.json
+site/packge-log.json
```

Comment:
Can't those just live inside the site folder? Inside another `.gitignore`

Reply:
Agreed, and a couple of points:

There's no need to put `node_modules` in `site/.gitignore` because it's covered by the root `.gitignore`.

Also, I think there's a typo in `site/packge-log.json`, I'm guessing it's supposed to be `site/package-lock.json`. That one is also unnecessary because it's covered by root `.gitignore`.

So `site/.gitignore` should have:

```
public
db.json
!/yarn.lock
```

The `!/yarn.lock` is so that `yarn.lock` of the site is not ignored, since we're not managing the site as part of the workspace.... though I'm starting to wonder if it should be part of the workspace.

Reply:
Will update!